cpp/CMakeLists.txt

```diff
@@ -1,6 +1,11 @@
 cmake_minimum_required (VERSION 2.8)
```

Comment:
Same as for the ``FindMlpack.cmake`` file; for more information take a look at: https://github.com/mlpack/mlpack/issues/870, let me know what you think.